src/workers.jl

```diff
@@ -105,6 +106,23 @@ function watch_and_terminate!(w::Worker, ev::Threads.Event)
     true
 end
 
+# Send signal to the given `Worker` process to trigger a profile.
+# Users can customise this profiling in the usual way, e.g. via
+# `JULIA_PROFILE_PEEK_HEAP_SNAPSHOT`, `Profile.set_peek_duration`, `Profile.peek_report[]`
+# See https://docs.julialang.org/en/v1/stdlib/Profile/#Triggered-During-Execution
+function trigger_profile(w::Worker, from::Symbol=:manual)
+    if !Sys.iswindows()
```

Comment:
why not windows?  because there isn't a SIGINFO equivalent?

Reply:
Good question! Looks like i did this because upstream Julia doesn't support this functionality on Windows... but idk why not